docs/dev-setup-non-vagrant.md

```diff
@@ -51,6 +51,8 @@ Install the following non-Python dependencies:
  * rabbitmq-server
  * libldap2-dev
  * python-dev
+ * python3-dev
```

Comment:
@pR0Ps the package list here is for `APT_DEPENDENCIES` (https://github.com/zulip/zulip/blob/master/tools/lib/provision.py#L128 note: I think it is worth adding in another PR to update `APT_DEPENDENCIES` to have Debian), so putting python3-dev and python-virtualenv are redundant with the later `apt-get install` where you have just added another python3-dev and python-virtualenv.